src/main/java/org/jenkinsci/gradle/plugins/jpi/server/JenkinsServerTask.java

```diff
@@ -118,6 +118,10 @@ public void execute(JavaExecSpec s) {
                     s.setMain(getExtractedMainClass().get());
                 }
                 s.args("--httpPort=" + getPort().get());
+                s.jvmArgs("--add-opens=java.base/java.lang=ALL-UNNAMED");
```

Comment:
🤔 Could you provide a comment why this is necessary? Shouldn't newer Jenkins versions run on Java 11 or later without the add opens?

Reply:
The maven plugin pom [does it](https://github.com/jenkinsci/plugin-pom/blob/master/pom.xml#L65-L66). It's mentioned [in the dev docs](https://github.com/jenkinsci/jenkins/blob/master/CONTRIBUTING.md?plain=1#L48), and we've had to do it to run test classes.

If it's not required that's great (was there an announcement?), but this was required at the time we upgraded to JDK 17.